monitors/SyntheticPegMonitor.js

```diff
@@ -38,10 +38,10 @@ class SyntheticPegMonitor {
     const defaultConfig = {
       deviationAlertThreshold: {
         // `deviationAlertThreshold`: Error threshold used to compare observed and expected token prices.
-        // if the deviation in token price exceeds this value an alert is fired.
+        // if the deviation in token price exceeds this value an alert is fired. If set to zero then fire no logs.
         value: 0.2,
         isValid: x => {
-          return x < 100 && x > 0;
+          return x < 100 && x >= 0;
```

Comment:
We don't want to track deviation off peg in UMIP-5. Rather than adding an override, it was way easier (and fewer code changes) to simply enable a value of 0 as a param here.